### PR TITLE
Added WFS and OGC Filter utils

### PR DESCRIPTION
**Pull request by weskamm**
This PR adds some util methods to work with WFS and OGC Filters.
Please review

### CODE REVIEW

**Comment by annarieger** [comment]
+1 from me as I know well where it comes from :sunglasses:

**Comment by hwbllmnn** [comment]
Nice! Just some minor comments

File: src/util/WFS.js [hunk 1]
```diff
@@ -0,0 +1,621 @@
...
+         * Captured parts are as follows:
+         *
+         * * Index `0`: Complete match.
+         * * Index `1`: `<ogc:Filter>` start tag including attributes, if any.
+         * * Index `2`: Anything between `<ogc:Filter>` start and end tag.
+         * * Index `3`: `</ogc:Filter>` end tag.
+         *
+         * Hat-tip: http://www.regular-expressions.info/examples.html
+         */
+        reMatchFilter: /(<ogc:Filter\b[^>]*>)(.*?)(<\/ogc:Filter>)/,
```
[the remark this answers is not in the document]
Comment (weskamm) [reply src/util/WFS.js]: will leave this for now, as this requires major changes

**Comment by weskamm** [comment]
Thanks for review, will merge now

**Pull request merged**